modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-an-variable.adoc

```diff
@@ -55,9 +32,17 @@ data:
   mykey: myvalue
 ----
 +
-This results in the environment variable named `FOO_ENV` and the value `myvalue` being provisioned into all workspace containers.
-+
-If the Secret provides more than one key, the environment variable name must be provided for each of the data keys as follows:
+In this example, the `FOO_ENV` environment variable and `myvalue` key value are provisioned into all workspace containers as follows:
+
+** The secret's environment variable, `env-name`, is declared as `FOO_ENV` in the `annotations`.
+
+** The `automount-workspace-secret` annotation with a boolean value defines the default secret mounting behavior (that can be overridden in a devfile):
+
+*** `'true'` enables automatic mounting into all workspace containers.
```

Comment:
Do these two values need to be in quotation marks?

Reply:
They have single quotes in the example code blocks provided here.